### PR TITLE
GH3098: Update supported RHEL version

### PR DESCRIPTION
**Pull request by apinnick**
Bug fix

Fixes issue # [3098](https://github.com/oVirt/ovirt-site/issues/3098)

Changes proposed in this pull request:

- Add attribute for latest supported RHEL version so that it is not hard-coded into the docs

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/main/CONTRIBUTING.md): @apinnick 

This pull request needs review by: @sandrobonazzola

### CODE REVIEW

File: source/documentation/common/collateral_files/_attributes.adoc [hunk 1]
```diff
@@ -108,4 +109,5 @@ ifdef::ovirt-doc[]
 :org-fullname: oVirt
 :container-platform: OKD
 :container-platform-virt: KubeVirt
+:supported-rhel-version: 9
```
Comment (sandrobonazzola) [inline source/documentation/common/collateral_files/_attributes.adoc:112]: I would say 8.7 or newer

**Review by sandrobonazzola — CHANGES_REQUESTED** [review]
See inline comment

**Comment by emarcusRH** [comment]
For RHV 4.4 SP1, there will never be another version of RHEL supported - the last and latest is supposed to be RHEL 8.6 EUS, so introducing an attribute for the latest RHEL version here is not a good idea for any RHV downstream docs

**Comment by apinnick** [comment]
> For RHV 4.4 SP1, there will never be another version of RHEL supported - the last and latest is supposed to be RHEL 8.6 EUS, so introducing an attribute for the latest RHEL version here is not a good idea for any RHV downstream docs

@emarcusRH See the GitHub issue. The problem is that Ovirt supports 8.7 and later. So we need an attribute.

**Review by sandrobonazzola — APPROVED** [review]
LGTM

**Pull request merged**